api_myportal.yaml

```diff
@@ -0,0 +1,52 @@
+swagger: "2.0"
+info:
+  version: 1.0.0
+  title: MyPortal API
+  description: MyPortal API for user authentication.
+host: localhost
+basePath: /myportal/api/v1
+schemes:
+  - https
+security:
+  - Token: []
+paths:
+  "/user":
+    x-swagger-router-controller: MyPortalController
+    get:
+      operationId: getUser
+      summary: Get user's data
+      description: Returns the user data needed by MyPortal backend.
+      responses:
+        "200":
```

Comment:
@BurnedMarshal 
Where is the "not found" case?

Reply:
This API will use only the data stored in Redis. If the token is missing we are into the `401` scenario. `404` response occurs when the user profile is missing but it doesn't happen into this API.